src/app/components/header/header.component.html

```diff
@@ -96,39 +97,93 @@
                     Deposited in UDC
                 </span>
             </div>
-            <ng-container
-                *ngFor="
-                    let token of tokens$ | async | slice: 0:2;
-                    trackBy: tokenTrackByFn
-                "
-            >
-                <div class="divider"></div>
-                <div fxLayout="column" fxLayoutGap="4px">
-                    <span>
-                        <span
-                            *ngIf="
-                                token.balance
-                                    | decimal: token.decimals as tokenBalance
-                            "
-                            [matTooltip]="tokenBalance"
-                            class="header__balance"
-                        >
-                            {{ tokenBalance | displayDecimals: 3 }}
+            <ng-container *ngIf="tokens$ | async as tokens">
+                <ng-container
+                    *ngFor="
+                        let token of tokens
+                            | slice: 0:(tokens.length > 2 ? 1 : 2);
```

Comment:
How are those tokens sorted? Can we somehow prioritize tokens with a higher general interest like DAI or so?

Reply:
The `TokenPollingService` already returns the tokens sorted. First by usage (times that the user transferred it), then by balance.